examples/redshiftserverless/resourcepolicy.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: redshiftserverless.aws.upbound.io/v1beta1
+kind: ResourcePolicy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: redshiftserverless/v1beta1/resourcepolicy
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    policy: |-
+      ${jsonencode({
+          Version = "2012-10-17"
+          Statement = [{
+            Effect = "Allow"
+            Principal = {
+              AWS = ["${data.aws_peer_account_id}"]
+            }
+            Action = [
+              "redshift-serverless:RestoreFromSnapshot",
+            ]
+            Sid = ""
+          }]
+        })}
```

Comment:
I think it would be better to use [this format](https://github.com/upbound/provider-aws/blob/5edbe14ab12d58ef4df1b484d58e016b4d65a343/examples/iam/policy.yaml#L11) for policy. What do you think?